src/seedu/addressbook/data/person/Address.java

```diff
@@ -55,4 +55,9 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+
+	@Override
+	public String getPrintableString() {
+		return (isPrivate) ? "Address: (private) ":"Address: " + this.value;
```

Comment:
Good use of ternary operator here, very handy. You can even shorten this statement further!